### PR TITLE
docs: Add Note About Apple Silicon

### PR DESCRIPTION
**Pull request by say25**
#### Description of Change

Add documentation about :apple: Apple Silicon. This may need to be backported to the 11.0.0 release notes. (Implemented in https://github.com/electron/electron/pull/24545) @MarshallOfSound 

#### Checklist

- [ ] PR description included and stakeholders cc'd
- [ ] `npm test` passes
- [ ] tests are [changed or added](https://github.com/electron/electron/blob/master/docs/development/testing.md)
- [x] relevant documentation is changed or added
- [ ] [PR release notes](https://github.com/electron/clerk/blob/master/README.md) describe the change in a way relevant to app developers, and are [capitalized, punctuated, and past tense](https://github.com/electron/clerk/blob/master/README.md#examples).

#### Release Notes

Notes:  Update documentation about Apple Silicon.

### CODE REVIEW

File: docs/tutorial/support.md [hunk 1]
```diff
@@ -96,13 +96,15 @@ Following platforms are supported by Electron:
...
 
+Electron 11.0.0 and later adds native support for Apple Silicon (`arm64`) devices.
+
 ### Windows
 
 Windows 7 and later are supported, older operating systems are not supported
 (and do not work).
 
 Both `ia32` (`x86`) and `x64` (`amd64`) binaries are provided for Windows.
-[Electron 6.0.8 and later add native support for Windows on Arm (`arm64`) devices](windows-arm.md).
```
Comment (codebytere) [inline docs/tutorial/support.md:105]: this was correct as-is - "Electron 6.0.8 and later" is a proxy for "versions" which as a plural subject require pluralization of the verb. To be more clear, let's instead say "include" here.

File: docs/tutorial/support.md [hunk 2]
```diff
@@ -96,13 +96,15 @@ Following platforms are supported by Electron:
 Only 64bit binaries are provided for macOS, and the minimum macOS version
 supported is macOS 10.10 (Yosemite).
 
+Electron 11.0.0 and later adds native support for Apple Silicon (`arm64`) devices.
```
Comment (codebytere) [inline docs/tutorial/support.md:99]: Same as below - let's say:
> Electron 11.0.0 and later include native support for Apple Silicon (`arm64`) devices.

**Review by codebytere — CHANGES_REQUESTED** [review]

File: docs/tutorial/support.md (hunk 1 above)
Comment (ckerr) [reply docs/tutorial/support.md:105]: At the risk of being pedantic... IMO this doesn't improve the grammar as "include" + "includes" have the same potential confusion as "add" and "adds". Moreover, "include" is slightly less clear than "add" in these sentences.

How about:

> Native support for Apple Silicon (`arm64`) devices was added in Electron 11.0.0.

> Native support for Windows on Arm (`arm64`) devices was added in Electron 6.0.8.

**Review by codebytere — APPROVED** [review]

**Review by ckerr — CHANGES_REQUESTED** [review]
At the risk of being pedantic... IMO this doesn't improve the grammar as "include" + "includes" have the same potential confusion as "add" and "adds". Moreover, "include" is slightly less clear than "add" in these sentences.

How about:

> Native support for Apple Silicon (`arm64`) devices was added in Electron 11.0.0.

> Native support for Windows on Arm (`arm64`) devices was added in Electron 6.0.8.

**Comment by say25** [comment]
@ckerr I think your suggestion makes a lot of sense, applied.

**Review by ckerr — APPROVED** [review]
@say25 thanks!

**Comment by ckerr** [comment]
`build-mac` and `appveyor: win-x64-testing-pr` CI failures are unrelated to the PR

**Review by miniak — APPROVED** [review]

**Pull request merged**